src/main/java/seedu/address/MainApp.java

```diff
@@ -138,18 +138,10 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
         ModelManager modelManager = new ModelManager(initialAddressBook, initialTaskBook, initialOrderBook, userPrefs);
         try {
             modelManager.checkClientAndOrderRelation();
-        } catch (DataConversionException e) {
-            logger.info("Data file not corrupted. Will be starting with a empty OrderBook");
-            initialOrderBook = new OrderBook();
-            modelManager = new ModelManager(initialAddressBook, initialTaskBook, initialOrderBook, userPrefs);
-        }
-
-        try {
             modelManager.checkTaskAndOrderRelation();
         } catch (DataConversionException e) {
-            logger.info("Data file not corrupted. Will be starting with a empty TaskBook");
-            initialTaskBook = new TaskBook();
-            modelManager = new ModelManager(initialAddressBook, initialTaskBook, initialOrderBook, userPrefs);
+            logger.warning("Data file not corrupted. Will be starting with a empty Data.");
```

Comment:
What does 'data file not corrupted mean` mean?

Reply:
Ok yeah, I will make the comment more clear!!